cmd/chart-up/main.go

```diff
@@ -0,0 +1,284 @@
+package main
+
+import (
+	"bytes"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io"
+	"io/fs"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/urfave/cli/v2"
+	"k8s.io/helm/pkg/chartutil"
+)
+
+var (
+	version    string
+	gitSHA     string
+	errNoChart = errors.New("no chart provided")
+)
+
+const (
+	FilePackage = "package.yaml"
+	FileChart   = "Chart.yaml"
+	FileValues  = "values.yaml"
+)
+
+func main() {
+	app := &cli.App{
```

Comment:
Can we simplify this and use the flags package like the other 2 utils in this repo? It'd be beneficial to the users to have a consistent ux.

Reply:
Sure, will do. Follow up question: I _like_ having the CHARTS env variable that overrides the flag with the same name, but do you think is that of any value?

Reply:
I ended up keeping the CHARTS env var as fallback value. Let me know your thoughts about it.